tenable/io/v3/was/user_templates/schema.py

```diff
@@ -0,0 +1,36 @@
+'''
+User Templates API Endpoint Schemas
+'''
+from marshmallow import Schema, fields, validate
+
+
+class PermissionSchema(Schema):
+    '''
+    Permission Schema for user-tempaltes
```

Comment:
Typo
Permission Schema for User-Templates

Reply:
fixed